api/systemIntegrations/put.js

```diff
@@ -265,6 +270,35 @@ function _postProvider(bag, next) {
   );
 }
 
+function _genSshKeys(bag, next) {
```

Comment:
Why is this necessary during the PUT? If the keys were not generated, the integration would not have been created to begin with, right?

Reply:
ok, removing it.